bindings/rust/standard/bench/Cargo.toml

```diff
@@ -20,7 +20,7 @@ pprof = { version = "0.14", features = ["criterion", "flamegraph"] }
 # debugging interop failures
 env_logger = "0.10"
 log = "0.4"
-symbolic-common = { version = "=12.13.4" } # newer versions require rust 1.82.0, see https://github.com/aws/s2n-tls/issues/5165
+symbolic-common = { version = "=12.13.4" }
```

Comment:
Wait, you didn't unpin this-- you just removed the comment. You need to actually stop pinning to a specific version.